code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -34,11 +33,41 @@
 	else
 		H.equip_or_collect(new H.species.survival_gear(H.back), slot_in_backpack)
 
-	var/religion_name = "Christianity" //Default
-	var/deity_name = "Space Jesus" //Also default, set below
+	var/obj/item/weapon/storage/bible/B // Initialised here because we might need it eslewhere
 
 	spawn(0) //We are done giving earthly belongings, now let's move on to spiritual matters
+		var/new_religion = sanitize(stripped_input(H, "You are the crew's Religious Services Chaplain. What religion do you follow and teach? (Please put your ID in your ID slot to prevent errors)", "Name of Religion", chap_religion.name), 1, MAX_NAME_LEN)
+		if(!new_religion)
+			new_religion = chap_religion.name // If nothing was typed
+		new_religion = lowertext(new_religion)
+
+		var/datum/job/J = H.mind.role_alt_title
+
+		for (var/R in typesof(/datum/religion))
+			var/datum/religion/rel = new R
+			var/halt = FALSE
+			for (var/key in rel.keys)
+				to_chat(world, "[key]")
+				if (new_religion == key)
+					rel.misc(H) // We do the misc things related to the religion
+					B = new rel.bible_type
+					B.name = rel.bible_name
+					B.deity_name = rel.deity_name
+					H.put_in_hands(B)
+					if (H.gender == FEMALE)
```

Comment:
You can do this with a ternary statement if you care to

Reply:
`J = (H.gender == FEMALE ? rel.female_adept : rel.male_adept)`
not sure the parens would be necessary
it's not a thing you *need* to do or even necessarily *should*, but you *can*

Reply:
I didn't even knew about ternary operators before
I'm always happy to learn